Week-2/Assignment 1/artist.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>
+            Book An Artist
+        </title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+
+    <body>
+      <div class="containerDiv">
+        <center><h1>Book An Artist For Events And Functions</h1></center>
+
+            <div class="bodyDiv" >
+                <div class="leftDiv">
+                        <p class="p1">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis id sodales metus. Mauris rhoncus blandit eleifend. 
+                            Aenean pharetra lorem nec ligula feugiat, aliquam consequat risus malesuada. 
+                            Phasellus malesuada dolor quis consequat sollicitudin. Curabitur lectus magna, posuere eget euismod quis, pretium ac risus.
+                            Ut et lacus ac ligula gravida ultrices eu sed ipsum.        
+                        </p>
+
+                    <img src="Mic.jpg" alt="Microphone">    
+                </div>
+
+                <div id="rightDiv">
+                    <p>Suspendisse sodales, nisl sit amet tempus porta, lacus ante dapibus libero, in tempus sem leo et mi.
+                       Phasellus hendrerit pellentesque eleifend. Nam egestas turpis erat, quis fermentum velit sodales sodales.
+                       Maecenas ornare, massa et imperdiet dapibus, risus nulla dapibus neque, tempus congue velit ligula eu mi.
+                    </p>
+
+                    <div>
+                        <h4>LOREM IPSUM DOLOR SIT</h4>
+                        <p>Aenean in lectus orci. Nulla finibus efficitur tellus ut placerat. Morbi volutpat luctus nibh, 
+                           sit amet iaculis lacus facilisis ac. Duis consectetur mi sit amet lorem vulputate, id volutpat ligula finibus.
+                           Duis pharetra ornare sem quis interdum. Aliquam id tortor gravida, dignissim mi ac, auctor felis.
+                        </p>
+                    </div>
+
+                    <div>
+                        <h4>LOREM IPSUM DOLOR SIT</h4>
+                        <p>Nulla hendrerit, ex ut ullamcorper vehicula, nunc lacus vehicula ligula, eget dapibus nisi mi at elit. 
+                           Integer facilisis libero a dapibus vehicula. Donec odio lectus, tristique ac tincidunt nec, varius sit
+                           amet ex. Nam blandit tellus et neque laoreet, ac egestas urna iaculis. Mauris vitae justo tempor, condimentum justo nec, viverra odio.
+                        </p>
+                    </div>
+
+                    <div id="formDiv">
+                        <form>
+                            <label for="fname">First name:</label><input type="text" id="fname" name="fname" placeholder="Mayank"><br>
+                            <label for="fname">Last name:</label><input type="text" id="fname" name="fname" placeholder="Sharma"><br>
```

Comment:
do not use id use class for css

Reply:
Ok. i will make the change and commit again.